Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -41,14 +42,16 @@ public ChromeBrowserFactory ([NotNull] IChromeConfiguration chromeConfiguration)
       _chromeConfiguration = chromeConfiguration;
     }
 
-    public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfrastructureConfiguration)
+    public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfrastructureConfiguration, DriverOptions options)
     {

```

Comment:
This PR is a little tricky. At this stage, we could assume that `options` have to be `NotNull`, since we could theoretically instance them one level above in `WebTestHelper`, we'd have the `ITestInfrastructure` we need for that, and carrying null around could be considered a code smell. On the other hand, having optional `DriverOptions` at `IBrowserFactory` level makes a lot of sense as well and would be analogous to `WebTestActionOptions`. I'm starting to think that the overall design is suboptimal, as all these little gripes might suggest a bigger design flaw. I dunno man ¯\_(ツ)_/¯